boxes.yaml.example

```diff
@@ -44,6 +44,7 @@ centos7-bats-ci:
       bats_tests:
         - "swapfile.bats"
         - "fb-install-katello.bats"
```

Comment:
That's the problem I have with this repo: I don't know how people use it. Maybe @ehelms or @stbenjam know it better.